### PR TITLE
remove old overview infrastructure and add home tab doc

### PR DESCRIPTION
**Pull request by Ancairon**
##### Summary

@netdata/product I don't want to make one giga PR, so let's merge this, essentially removing a super old doc and adding a missing document about the Home tab.

I will follow up with the PR on learn regarding the removal of the old document after this is merged.

(I will update the docs inside the section and then update the parent overview page)

### CODE REVIEW

**Review by shyamvalsan — APPROVED** [review]

File: docs/dashboard/home-tab.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- Connection stability (e.g. Stable, Unstable)
+- Replication factor (e.g. None, Single)
+
+## Data replication
+
+There are two views about data replication in the Home tab:
+
+The first bar chart presents the amount of **Parents**, **Children** and **Standalone** nodes.
+
+The second bar chart presents the amount of nodes depending on their Replication factor, **None**, **Single** and **Multi**.
```
Comment (ilyam8) [inline docs/dashboard/home-tab.md:40]:
```suggestion
There are two views about data replication in the Home tab:

The first bar chart presents the amount of **Parents**, **Children** and **Standalone** nodes.

The second bar chart presents the number of nodes depending on their Replication factor, **None**, **Single** and **Multi**.
```

File: docs/dashboard/home-tab.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Home Tab
+
+The Home tab allows users to see an overview of their Space.
+
+## Total nodes
+
+The total amount of nodes is presented, dissected also by their state, Live, Offline or Stale.
```
Comment (ilyam8) [inline docs/dashboard/home-tab.md:7]:
```suggestion
The total number of nodes is presented and dissected by their state, Live, Offline or Stale.
```

File: docs/dashboard/home-tab.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+## Active alerts
+
+The number of the active alerts are presented in a donut chart, while also having counters for both Critical and Warning alerts.
```
Comment (ilyam8) [inline docs/dashboard/home-tab.md:11]:
```suggestion
The number of active alerts is presented in a donut chart, while also having counters for both Critical and Warning alerts.
```

File: docs/dashboard/home-tab.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+There are two views about data replication in the Home tab:
+
+The first bar chart presents the amount of **Parents**, **Children** and **Standalone** nodes.
+
+The second bar chart presents the amount of nodes depending on their Replication factor, **None**, **Single** and **Multi**.
+
+## Alerts overview over the last 24h
+
+There are two views that displays information about nodes that produced the most alerts and top alerts in the last 24h.
```
Comment (ilyam8) [inline docs/dashboard/home-tab.md:44]:
```suggestion
There are two views that display information about nodes that produced the most alerts and top alerts in the last 24 hours.
```

File: docs/dashboard/home-tab.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+In the Home tab there is a shortcut button in order to start an instant conversation with the [Netdata Assistant](https://github.com/netdata/netdata/edit/master/docs/cloud/netdata-assistant.md).
+
+## Space metrics
+
+There are three key metrics that are displayed in the Home tab, **Metrics collected**, **Charts visualized** and **Alerts configured**.
+
+## Data retention per Nodes
+
+This bar chart presents the amount of nodes in relation to their retention period.
```
Comment (ilyam8) [inline docs/dashboard/home-tab.md:60]:
```suggestion
This bar chart shows the number of nodes based on their retention period.
```

**Review by ilyam8 — DISMISSED** [review]

File: docs/dashboard/home-tab.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
+# Home Tab
+
+The Home tab allows users to see an overview of their Space.
```
[the remark this answers is not in the document]
Comment (Ancairon) [reply docs/dashboard/home-tab.md:3]: good catch

File: docs/dashboard/home-tab.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+The map classification can be altered, allowing the categorization of nodes by:
+
+- Status (e.g. Live)
+- OS (e.g. Ubuntu)
+- Technology (e.g. Container)
+- Agent version (e.g. X.XX.X)
+- Replication factor (e.g. Single, Multi)
+- Cloud provider (e.g AWS)
+- Cloud region (e.g. Europe)
```
Comment (hugovalente-pm) [inline docs/dashboard/home-tab.md:25]:
```suggestion
- Cloud region (e.g. us-east-1)
```

**Review by ilyam8 — DISMISSED** [review]

File: docs/dashboard/home-tab.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+## Nodes map
+
+A map consisting of node entries allows for quick hoverable information about each node, while also presenting node status in a color-coded way.
+
+The map classification can be altered, allowing the categorization of nodes by:
+
+- Status (e.g. Live)
+- OS (e.g. Ubuntu)
+- Technology (e.g. Container)
+- Agent version (e.g. X.XX.X)
```
Comment (hugovalente-pm) [inline docs/dashboard/home-tab.md:22]:
```suggestion
- Agent version (e.g. v1.45.2)
```

**Review by hugovalente-pm — APPROVED** [review]

**Pull request merged**